CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ endif(CMAKE_SOURCE_DIR STREQUAL CMAKE_BINARY_DIR)
 
 project(pgmoneta VERSION ${VERSION_STRING} LANGUAGES  C)
 
+set (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
+set (CMAKE_LINKER_FLAGS_DEBUG "${CMAKE_LINKER_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
+
```

Comment:
Those are in `src/CMakeLists.txt` for `clang`

Reply:
I have removed it, Thanks.